src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1600,28 +1603,52 @@ void TableDockWidget::showHeatMap() {
   }
 }
 
+void TableDockWidget::editSelectedPeakGroup()
+{
+  if (treeWidget->selectedItems().size() != 1)
```

Comment:
Does it count a parent+children selection as 1 selected item or not? Asking here since I'm unable to test atm.

Reply:
This is a tree-view level check, so I am guessing it has to be at most one item selection. It will be true if one parent item is selected (with or without the children expanded). But I double check just to be sure.